src/components/SearchName/Search.js

```diff
@@ -68,55 +68,52 @@ const SearchForm = styled('form')`
   }
 `
 
-class Search extends React.Component {
-  state = {
-    type: null,
-    filterOpen: false
-  }
-  handleParse = () => {
-    const type = parseSearchTerm(this.input.value)
-    this.setState({ type })
+function Search({ history, className, style }) {
+  const [type, setType] = useState(null)
+  //const [filterOpen, setFilterOpen] = useState(false)
```

Comment:
Do you still need this one?

Reply:
Not right now. Was debating whether to remove it, but I have already built the Filter components almost ready to add in if/when we want to add them in. So not sure if I should remove them all entirely or leave it commented out ready to go